src/main/resources/services.bom.json

```diff
@@ -489,6 +489,26 @@
           "flow": "inbound"
         }
       ]
+    },
+    {
+      "bom-ref": "service:com.aquasec",
+      "provider": {
+        "name": "Trivy",
+        "url": [
+          "https://www.aquasec.com/products/trivy"
+        ]
+      },
+      "group": "com.aquasec",
+      "name": "Trivy",
+      "description": "Trivy is a comprehensive and versatile security scanner. Trivy has scanners that look for security issues, and targets where it can find those issues.",
+      "authenticated": true,
+      "x-trust-boundary": true,
+      "data": [
+        {
+          "classification": "public",
+          "flow": "inbound"
+        }
+      ]
```

Comment:
This implies that Dependency-Track reaches out to `https://www.aquasec.com/products/trivy`, which is not the case, since Trivy is self-hosted. This should be removed.